### ISSUE TITLE
plugin to reorder slides on slide editor

### ISSUE THREAD

**Issue by psenough** [Gargaj/wuhu#68]
Would be nice quality of life improvement to be able to reorder the slides on the slide editor page instead of having to login to FTP and rename them manually.

Most trivial implementation would be to just have a box to rename them when you're editing them.
Or drag and drop the slides in order that automagically handles the renaming for you.

**Comment by chainq** [comment]
I would love this feature too. However, it's a bit tricky with the autogenerated slides (timetable, oneliner), I suppose?

**Comment by Gargaj** [comment]
Not necessarily, cos the slides are served as a JSON/XML (can't remember) so there can be metadata/ordering.
